### PR TITLE
Golden Globe API week 13

### PR DESCRIPTION
**Pull request by kathinka**
Render Link [Render](https://golden-globe-api.onrender.com/)

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,30 +1,240 @@
...
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// Start with showing the updated documentation for the api
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(docu);
 });
```
Comment (AntonellaMorittu) [inline server.js:18]: consider a dynamic approach using `expressListEndpoints` :

```javascript
app.get("/", (req, res) => {
  const endpoints = expressListEndpoints(app);
  res.json(endpoints);
}); 
```

File: server.js
Comment (AntonellaMorittu) [inline server.js:1]: please format your files before committing 👍

File: documentation.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  },
+  {
+    "path": "/movies",
+    "methods": [
+      "GET"
+    ],
+    "middlewares": [
+      "anonymous"
+    ]
+  },
```
Comment (AntonellaMorittu) [inline documentation.json:19]: To be more descriptive in your api usage, you could add the queries to filter the data:

```json
 "middlewares": [
    "anonymous"
   ],
  "query_params": {
    "category": "Filter movies by category",
    "year": "Filter movies by year",
    "win": "Filter movies by win status (true/false)",
    "nominee": "Filter movies by nominee",
    "film": "Filter movies by film or series name"
  }
```

**Review by AntonellaMorittu — APPROVED** [review]
Hello Kathinka, well done completing your first API! It meets the main requirements, I left some comments in the code for you to read and improve ⭐ ciao! 👋

File: server.js
Comment (kathinka) [reply server.js:1]: Still doesn't understand  as I have set this to two spaces everywhere. I sendt you a scshot in slack so you'll see how it looks.

File: documentation.json (hunk 2 above)
Comment (kathinka) [reply documentation.json:19]: will do!

**Pull request closed without merging**